### ISSUE TITLE
replace encrypt_password to hash_password

### ISSUE THREAD

**Issue by jwag956** [jwag956/flask-security#51]

**Comment by michaelbukachi** [comment]
I'd like to take this up.

**Comment by jwag956** [comment]
You are certainly welcome to - I will review and merge as appropriate.

However (there is always a however) - I am still in the learning stage which is why I made this fork and created some releases. I am working on creating 3.1 which has almost all of the current PRs from the upstream repo. before I do that - I intend to reach out to the current maintainers and see if we can come to an agreement on moving flask-security forward. 2 independent forks isn't ideal - it would be great if we can agree early to move the main branch forward.

Bottom line - if you send a PR to my fork - it might well have to be redone in upstream if that's where it all ends up. 

I am hoping to resolve all this in May...

**Comment by michaelbukachi** [comment]
Awesome. I'll work on it but I'll hold the pull request till you've finished talking with the maintainers.

**Comment by baurt** [comment]
I think this issue was taken care of in https://github.com/jwag956/flask-security/pull/86
